lib/patch-series.ts

```diff
@@ -286,7 +287,14 @@ export class PatchSeries {
                             basedOn = match2[2];
                             break;
                         case "cc:":
-                            cc.push(match2[2]);
+                            addressparser(match2[2]).forEach((e) => {
+                                if (e.name) {
+                                    cc.push(encodeWords(e.name)
```

Comment:
Is it expected that `generateMBox()` also calls `encodeWords` ?

Reply:
Oh, I guess not... Good point. Will fix.

Reply:
Thanks, looks reasonable to me now (not that I have any experience to be sure, though)